### PR TITLE
[TeamItems] CAES-1201: Fix items decryption

### PR DESCRIPTION
**Pull request by sa715**
Signed-off-by: Sergey Alekseyev <sa@4xxi.com>

### CODE REVIEW

File: packages/common/sagas/workflow.js [hunk 1]
```diff
@@ -134,6 +171,7 @@ function* initPersonal(withDecryption) {
 
 function* initTeam(team, withDecryption) {
   try {
+    console.log('Init team');
```
Comment (cu4xxi) [inline packages/common/sagas/workflow.js:174]: консоль

File: packages/common/sagas/entities/item.js [hunk 2]
```diff
@@ -351,15 +391,27 @@ export function* createItemSaga({
...
-      yield call(Router.push, ROUTES.DASHBOARD);
+      if (!teamId && currentTeamId === TEAM_TYPE.PERSONAL) {
+        const systemItemData = yield call(
+          generateSystemItem,
+          ENTITY_TYPE.ITEM,
+          userPersonalDefaultListId,
+          itemId,
+        );
+        systemItemData.relatedItem = itemId;
+        console.log('System data');
```
Comment (beshkenadze) [inline packages/common/sagas/entities/item.js:410]: Please, remove the console calls.

File: packages/common/selectors/entities/item.js [hunk 3]
```diff
@@ -71,18 +71,37 @@ export const itemsChildItemsBatchSelector = createSelector(
...
+    Object.values(items).filter(({ type }) => type === ITEM_TYPE.SYSTEM) || [],
+);
+
+export const systemItemsBatchSelector = createSelector(
+  systemItemsSelector,
+  itemIdsPropSelector,
+  (systemItems, itemIds) =>
+    itemIds.map(
+      itemId => {
+//        console.log(systemItems);
```
Comment (beshkenadze) [inline packages/common/selectors/entities/item.js:83]: Please, remove the commented lines

**Review by beshkenadze — CHANGES_REQUESTED** [review]

File: packages/common/selectors/entities/item.js (hunk 3 above)
Comment (sa715) [reply packages/common/selectors/entities/item.js:83]: Done

File: packages/common/sagas/entities/item.js (hunk 2 above)
Comment (sa715) [reply packages/common/sagas/entities/item.js:410]: Done

**Review by beshkenadze — APPROVED** [review]

**Pull request merged**